Lib/platform.py

```diff
@@ -134,7 +134,11 @@
 
 # Directory to search for configuration information on Unix.
 # Constant used by test_platform to test linux_distribution().
-_UNIXCONFDIR = '/etc'
+try:
+    _UNIXCONFDIR = '/etc/os-release'
```

Comment:
Is `/etc/os-release` a directory in some distributives?

Reply:
/etc has os-release file in some linux distributions

Reply:
Naming a path of the regular file `_UNIXCONFDIR` looks confusing.